src/v/kafka/types.h

```diff
@@ -122,6 +122,14 @@ inline kafka::leader_epoch leader_epoch_from_term(model::term_id term) {
         return kafka::invalid_leader_epoch;
     }
 }
+
+/// Bytes accounted for by the kafka::usage_manager will not take into
+/// consideration data batches via produce/fetch requests to these topics
+const auto usage_excluded_topics = std::to_array(
+  {model::topic("_schemas"),
+   model::topic("__audit"),
+   model::topic("__redpanda_e2e_probe")});
```

Comment:
would this be better in a server specific file? like server.h or the usage manager.h stuff?